### ISSUE TITLE
[UWP][Accessibility] Middle pane is not accessible and focus moves to background Screen.

### ISSUE THREAD

**Issue by almedina-ms** [microsoft/AdaptiveCards#3831]
# Platform

* UWP

# Version of SDK

master

# Details

Repro Steps:
1. Launch the application.
2. New screen starts appearing. Navigate to the top header elements.
3. Now select "Host Config" button. It will update the bottom pane. Navigate them.
4. Verify the Issue

Actual Result: 
Middle pane is not accessible and focus moves to background Screen.
Screen reader is announcing, information about background screen.

Expected Result: 
Middle pane should be accessible and focus should not move to background Screen.
